code/game/jobs/job/security_vr.dm

```diff
@@ -13,6 +14,7 @@
 
 /datum/job/warden
 	pto_type = PTO_SECURITY
+	dept_time_required = 20
```

Comment:
Warden's not a sub-head though?

Reply:
Some jobs are unique in departments like QM and PF. They received the same treatment.

Reply:
Warden *is* a sub-head, underneath HoS, technically. Iirc.